src/components-v2/InfluenceVault/InfluenceVaultPerfomanceTab.tsx

```diff
@@ -137,15 +124,7 @@ const BveCvxPerformance = ({ vault }: Props): JSX.Element => {
 						<SpecItem
 							name={
 								<span>
-									% CVX Received from 10k{' '}
-									<Link
-										href="https://curve.fi/factory/52/"
-										target="_blank"
-										rel="noopener"
-										display="inline"
-									>
-										bveCVX swap
-									</Link>
+									<MarkupText text={info.perfomanceInfo.liquity[0]} onClick={handleLinkClick} />
```

Comment:
unless we're sure we'll always want the fist liquidity entry, I would probably store the liquidity in its own "liquidity" key in the config file

Reply:
fixed